flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/test/util/TestBaseUtils.java

```diff
@@ -26,6 +26,7 @@
 import akka.pattern.Patterns;
 import akka.util.Timeout;
 
+import com.google.common.base.Joiner;
```

Comment:
Avoid using Guava API calls.  You could use Commons lang StringUtils.join() instead.